Adds full code snippets to quickstart guides (#73)
kluster-ai/docs: get-started/start-building/real-time.md

In addition, you can see the complete list of available models programmatically using the [list supported models](/api-reference/reference/#list-supported-models){target=\_blank} endpoint.

## Quickstart snippets

> The following  code snippets provide a full end-to-end real-time inference example for different models supported by kluster.ai. You can simply copy the snippet and paste it into your local environment. 

Reviewer: ```suggestion
The following code snippets provide a full end-to-end real-time inference example for different models supported by kluster.ai. You can simply copy and paste the snippet into your local environment. 
```

### Python

To use these snippets, run the Python script and enter your kluster.ai API key when prompted.

> ??? example "DeepSeek R1"

Reviewer: Code instead of exmaple

    ```

??? example "LLama 3.1 8B"

    ```python
    --8<-- 'code/get-started/start-building/real-time/real-time-llama3.1-8.py'
    ```

??? example "LLama 3.1 405B"

    ```python
    --8<-- 'code/get-started/start-building/real-time/real-time-llama3.1-405.py'
    ```

??? example "LLama 3.3 70B"

    ```python
    --8<-- 'code/get-started/start-building/real-time/real-time-llama3.3-70.py'
    ```

### CLI

Similarly, the following curl commands showcase how to easily send a chat completion request to kluster.ai for the different supported models. This example assumes you've exported your kluster.ai API key as the variable `API_KEY`.

??? example "DeepSeek R1"

    ```bash
    --8<-- 'code/get-started/start-building/real-time/real-time-deepseekR1.md'
    ```
    
??? example "LLama 3.1 8B"

    ```bash
    --8<-- 'code/get-started/start-building/real-time/real-time-llama3.1-8.md'
    ```

??? example "LLama 3.1 405B"

    ```bash
    --8<-- 'code/get-started/start-building/real-time/real-time-llama3.1-405.md'
    ```

??? example "LLama 3.3 70B"

    ```bash
    --8<-- 'code/get-started/start-building/real-time/real-time-llama3.3-70.md'
    ```

## Real-time inference flow

> This section goes through the details a real-time inference process using kluster.ai API and the DeepSeek R1 model, but you can adapt it to any of the [supported models](#supported-models).

Reviewer: ```suggestion
This section details the real-time inference process using the kluster.ai API and DeepSeek R1 model, but you can adapt it to any of the [supported models](#supported-models).
```

### Submitting a request

The kluster.ai platform offers a simple, [OpenAI-compatible](/get-started/openai-compatibility/){target=\_blank} interface, making it easy to integrate kluster.ai services seamlessly into your existing system.

The following code shows how to do a chat completions request using the OpenAI library.

=== "Python"

    ```python
    --8<-- 'code/get-started/start-building/real-time/real-time-01.py:01:22'
    ```

> If successful, the `completion` variable contains full response, which you'll need to analyze to extract the answer you are looking for. In terms of configuration for real-time inferences, there several parameters that you need to tweak:

Reviewer: ```suggestion
If successful, the `completion` variable contains a full response, which you'll need to analyze to extract the answer you are looking for. In terms of configuration for real-time inferences, there are several parameters that you need to tweak:
```

}
```

> The following snippet demonstrate how you can extract the data, log it to the console, and save it to a JSON file.

Reviewer: ```suggestion
The following snippet demonstrates how to extract the data, log it to the console, and save it to a JSON file.
```

=== "Python"

    ```python
    --8<-- 'code/get-started/start-building/real-time/real-time-01.py:23:47'
    ```

> For a detailed breakdown of the chat completion object, see the [**chat completion API reference**](/api-reference/reference#chat-completion-object){target=\_blank} section.

Reviewer: chat completion API reference shouldn't be in bold

??? code "View the complete script"

    === "Python"

        ```python
        --8<-- 'code/get-started/start-building/real-time/real-time-01.py'
        ```

## Third-party integrations

> You can also set up a [third-party LLM integrations](/get-started/integrations/){target=\_blank} using the kluster.ai API. For step-by-step instructions, check out the following integration guides:

Reviewer: ```suggestion
You can also set up [third-party LLM integrations](/get-started/integrations/){target=\_blank} using the kluster.ai API. For step-by-step instructions, check out the following integration guides:
```
